src/main/java/de/tum/in/www1/artemis/service/metis/ConversationMessagingService.java

```diff
@@ -443,7 +443,7 @@ private Conversation mayUpdateOrDeleteMessageElseThrow(Post existingMessagePost,
      * @return list of similar posts
      */
     public List<Post> getSimilarPosts(Long courseId, Post post) {
-        PostContextFilter postContextFilter = new PostContextFilter(courseId);
+        PostContextFilter postContextFilter = new PostContextFilter(courseId, null, null, null, null, false, false, false, null, null);
```

Comment:
see above comment, another constructor/method could make the code more readable.